docs/howto-tsg.md

```diff
@@ -12,6 +12,7 @@
         - [Service-side logs](#add_logs_service)
 - [Serverless mode troubleshooting](#serverless_mode_tsg)
 - [Classic mode troubleshooting](#classic_mode_tsg)
+- [Service health](#service_health)
```

Comment:
`#service_health` link not work. Need add `<a>` link before the header in that section, or directly use header way like `#Service-health`